src/components/Blocking/ReportPostAnonymous.js

```diff
@@ -77,7 +78,7 @@ const ReportPostAnonymous = ({refReportPostAnonymous, onSelect, onSkip}) => {
       <ScrollView nestedScrollEnabled={true}>
         <TouchableOpacity style={styles.btnSkip} onPress={() => onSkip()}>
           <Text style={styles.btnSkipText}>Skip & just block this post</Text>
-          <IconFA5 name="chevron-right" size={17} color={'#000'} />
+          <IconFA5 name="chevron-right" size={17} color={COLORS.black} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job replacing the hardcoded color with a value from the theme. This improves consistency and maintainability.
```diff
- <IconFA5 name="chevron-right" size={17} color={'#000'} />
+ <IconFA5 name="chevron-right" size={17} color={COLORS.black} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->